README.md

```diff
@@ -1,2 +1,20 @@
 # price-feeder-inspector
 Script inspecting validator votes and missing currencies
+
+
+
+## Vote Inspector Python Script 
+This Python script fetches live submitted aggregated votes from validators and cross-checks them with oracle-accepted denominations. It displays the missing denominations of votes.
+
+### How to run ? 
+```python 
+## For Canon 
+$ python3 votes_inspector.py prod
```

Comment:
I think this is wrong. Also, let's use `--network` rather than positional argument, and have default setting to use `mainnet`.

Reply:
Ok